saleor/graphql/order/bulk_mutations/order_bulk_create.py

```diff
@@ -1233,178 +1576,248 @@ def create_single_fulfillment(
                 quantity=line_input["quantity"],
             )
             lines.append(OrderBulkFulfillmentLine(fulfillment_line, warehouse))
+            line_index += 1
 
         return OrderBulkFulfillment(fulfillment=fulfillment, lines=lines)
 
     @classmethod
     def create_single_order(
```

Comment:
The function is quite long (1584-1800). Have you considered adding some helper methods to make it shorter?

Reply:
Reduced it a bit (216 -> 130). Let me know if you have other ideas how to organise the whole module.